guides/source/caching_with_rails.md

```diff
@@ -512,6 +512,26 @@ class ProductsController < ApplicationController
 end
 ```
 
+Sometimes we want to cache response (static page) that never gets expired.
+To achieve that, we can use `http_cache_forever` helper and by doing so browser
+and proxies will cache it indefinitely.
+
+By default cached response will be private. To make it public response, pass
```

Comment:
`By default cached response will be private, cached only on the user's web browser.`